src/Fixer/PhpTag/BlankLineAfterOpeningTagFixer.php

```diff
@@ -85,8 +85,12 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
             $tokens[0] = new Token([$token->getId(), rtrim($token->getContent()).$lineEnding]);
         }
 
-        if (!$tokens[1]->isWhitespace() && false === strpos($tokens[1]->getContent(), "\n")) {
-            $tokens->insertAt(1, new Token([T_WHITESPACE, $lineEnding]));
+        if (false === strpos($tokens[1]->getContent(), "\n")) {
```

Comment:
Is it OK to check only fofr `\n` instead of RegEx with `\R`?

Reply:
we support "\n" and "\r\n" but not "\r" only (old Mac thing I think?), so I think this is fine (we do this on other places as well)